Makefile

```diff
@@ -416,6 +420,18 @@ ifeq ($(RUN_MODE),kubernetes)
 	kubectl -n $(KUBE_NAMESPACE) scale deploy/argo-server --replicas 1
 endif
 
+# nuke is like "clean" but attempts to return you to a state as if it had never been installed at all
+# this only available in "safe" local development
+ifeq (true,$(filter true, $(DOCKER_DESKTOP) $(K3D)))
```

Comment:
`k3d` is not guaranteed to be a dev env, but if you're using `make nuke` you better know what you're doing.

Reply:
I hope that "nuke" help people be careful - but k3d should only be desktop - k3s might not